treeherder/autoclassify/matchers.py

```diff
@@ -318,10 +317,3 @@ def best_match(self, matches):
                 if best_match is None or new_ratio > best_match[0]:
                     best_match = (new_ratio, match)
         return best_match
-
-
-def register():
-    """Register matchers enabled in settings.AUTOCLASSIFY_MATCHERS."""
-    for obj_name in settings.AUTOCLASSIFY_MATCHERS:
```

Comment:
I think `AUTOCLASSIFY_MATCHERS` in settings.py is now unused after this

Reply:
Fixed and rebased in 970be29